kolibri/core/bookmarks/models.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import models
+from morango.models import UUIDField
+
+from kolibri.core.auth.models import FacilityUser
+
+
+class Bookmark(models.Model):
+
+    id = UUIDField(primary_key=True, default=uuid.uuid4)
+    content_id = models.CharField(max_length=50, null=False, blank=False)
+    contentnode_id = models.CharField(max_length=50, null=False, blank=False)
+    channel_id = models.CharField(max_length=50, null=False, blank=False)
+    facility_user = models.ForeignKey(FacilityUser, blank=False)
```

Comment:
For most other models we have used `user` as the name of the foreign key to a facility user, would be good to keep this consistent.